### ISSUE TITLE
The Defias Traitor

### ISSUE THREAD

**Issue by Ysangrim** [dalaranwow/dalaran-wow#4683]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The Defias Traitor http://www.wowhead.com/npc=467 seems stuck at the entrance of the Deadmines zone in westfall.

**Expected behaviour**: He should be at Sentinel hill allowing to start http://www.wowhead.com/quest=155/the-defias-brotherhood

**Steps to reproduce the problem**:

1. Reset him to Sentinel Hill.
2. https://youtu.be/XowGA0ZA81U?t=352 Starting the quest http://www.wowhead.com/quest=155/the-defias-brotherhood

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello, thank you for the report @Ysangrim .
This issue and few more are addressed here #2579
